### PR TITLE
add x264(+Yasm) as dep in ffmpeg easyconfig

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyconfigs/y/Yasm/Yasm-1.3.0-intel-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+name = 'Yasm'
+version = '1.3.0'
+
+homepage = 'http://www.tortall.net/projects/yasm/'
+description = """Yasm: Complete rewrite of the NASM assembler with BSD license"""
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.tortall.net/projects/yasm/releases/']
+
+toolchain = {'name': 'intel', 'version': '2015b'}
```
Comment (wpoely86) [inline easybuild/easyconfigs/y/Yasm/Yasm-1.3.0-intel-2015b.eb]: move up?

**Comment by wpoely86** [comment]
lgtm

File: easybuild/easyconfigs/y/Yasm/Yasm-1.3.0-intel-2015b.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/y/Yasm/Yasm-1.3.0-intel-2015b.eb]: damn :)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c36a72cf68ca98feccc8 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**